engine/src/main/java/org/hibernate/validator/internal/xml/ValidationBootstrapParameters.java

```diff
@@ -136,6 +142,15 @@ public void setClockProvider(ClockProvider clockProvider) {
 		this.clockProvider = clockProvider;
 	}
 
+	public Map<Key, ValueExtractorDescriptor> getValueExtractors() {
+		return valueExtractors;
+	}
+
+	public void addValueExtractor(ValueExtractor<?> valueExtractor) {
```

Comment:
I would pass the descriptor here. You're forced to unwrap it then rewrap it in 2 places.